libraries/AP_AHRS/AP_AHRS.cpp

```diff
@@ -832,8 +841,8 @@ bool AP_AHRS::airspeed_sensor_enabled(void) const
 bool AP_AHRS::_airspeed_estimate(float &airspeed_ret, AirspeedEstimateType &airspeed_estimate_type) const
 {
 #if AP_AIRSPEED_ENABLED && AP_GPS_ENABLED
-    if (airspeed_sensor_enabled()) {
-        uint8_t idx = get_active_airspeed_index();
```

Comment:
This function may then return another sensor which is not the **PRIMARY** so it could be set to not use, or unhealthy. Or conversely we could not be using a backup sensor because the primary is not healthy or not set to use.